website/docs/for-android/how-to/using-the-portals-plugin.md

```diff
@@ -0,0 +1,345 @@
+---
+title: How to Communicate with a Portal Web Application
+sidebar_label: Communicate with a Portal Web Application
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The `PortalsPlugin` provides useful features to aid in communication between your Web and Native applications. It is included in the Ionic Portals library by default and takes advantage of the Capacitor Plugin system.
+
+## Setup
+
+### Android and iOS
```

Comment:
Since this is in the android section do we want to remove reference to iOS in this